lib/models/item.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:uuid/uuid.dart';
+
+// Uuid to generate Ids for the todos
+Uuid uuid = const Uuid();
```

Comment:
```suggestion
const uuid = Uuid();
```

https://dart.dev/tools/linter-rules/prefer_const_declarations